packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -548,27 +548,38 @@
     </SF.Item>
   </Shw::Flex>
 
-  <Shw::Text::H4>States</Shw::Text::H4>
+  <Shw::Text::H3>States</Shw::Text::H3>
 
   <Shw::Flex as |SF|>
-    <SF.Item>
+    <SF.Item @label="Base">
```

Comment:
you can avoid this duplication using a wrapping `{{#let}}` declaration (see [an example here](https://github.com/hashicorp/design-system/blob/main/packages/components/tests/dummy/app/templates/components/accordion.hbs#L254)) and use the boolean to control if add the `@copyItemTitle` or not

Reply:
I probably didn't think about this since I was focused on copy/paste from other component pages, as our team has discussed many times before. I will integrate this improvement if there's time, or file a JIRA ticket to review this (if this component page needs it, there's a chance other preview pages do, too).